test/unit/jest.config.js

```diff
@@ -14,29 +14,23 @@
  * limitations under the License.
  */
 module.exports = {
-  roots: ['./'],
+  rootDir: '../..',
+  roots: ['./test/unit', './src'],
   moduleNameMapper: {
     // mock files that jest doesn't support like CSS and SVG files
     '\\.css$': './../module-mock.js',
     '\\.svg$': './../module-mock.js',
   },
   testMatch: ['**/?(*.)+(spec|test).[t]s'],
-  testPathIgnorePatterns: ['/node_modules/', 'dist'],
+  testPathIgnorePatterns: ['/node_modules/', 'dist', 'src'],
   transform: {
     '^.+\\.ts?$': 'ts-jest',
   },
   collectCoverageFrom: ['**/*.{ts,js}'],
-  coveragePathIgnorePatterns: ['/node_modules/', 'dist', 'test'],
-  coverageThreshold: {
-    global: {
-      branches: 80,
-      functions: 80,
-      lines: 80,
-      statements: 80,
-    },
-  },
```

Comment:
:question: I guess this disable the 'fail build' detection when we are under the threshold?

Reply:
Yes ^^